app/src/main.py

```diff
@@ -66,10 +67,11 @@ async def on_start(self):
         await self.Vehicle.OBD.Speed.subscribe(self.on_speed_change)
 
     # Is executed when receiving VehicleDataBroker signal.
-    async def on_speed_change(self, data):
+    async def on_speed_change(self, data: DataPointReply):
         logger.debug("Data received: %s", data)
         # Getting current speed from VehicleDataBroker.
```

Comment:
this comment is not accurate anymore.

please also consider removing / reworking the other comment lines, at the moment they only repeatwhat the code already tells the reader clearly.

Reply:
Fix as discussed.